browser/ui/webui/brave_rewards_ui.cc

```diff
@@ -598,10 +598,15 @@ void RewardsDOMHandler::OnGetAddresses(
       func_name == "addresses" || func_name == "addressesForPaymentId")) {
     base::DictionaryValue data;
     data.SetString("BAT", addresses.at("BAT"));
-    data.SetString("BTC", addresses.at("BTC"));
-    data.SetString("ETH", addresses.at("ETH"));
-    data.SetString("LTC", addresses.at("LTC"));
-
+    if (addresses.count("BTC") == 1) {
```

Comment:
could we just iterate through `addresses` map, so that we don't have `if`'s

Reply:
But we have hardcoded strings and wouldn't necessarily know which iteration gets set with which string.

Reply:
```
for (auto& address : addresses) {
  data.SetString(address.first,  address.second);
}
```

I was thinking something like this

Reply:
Ok. That works but FYI `SetString` is deprecated.

using:
```
base::Value data(base::Value::Type::DICTIONARY);
for (auto& address : addresses) {
  data.SetKey(address.first, base::Value(address.second))
}
```

Reply:
yeah I just copied what we have in the code now :) great

Reply:
Done.